libdrgn/program.c

```diff
@@ -50,6 +50,27 @@ drgn_program_platform(struct drgn_program *prog)
 	return prog->has_platform ? &prog->platform : NULL;
 }
 
+void drgn_program_init_language(struct drgn_program *prog)
```

Comment:
This is called every time we load debug info, so it needs an `if (prog->lang) return;`.

Also, this can be static.